migrations/versions/4039aa74290f_sqlalchemy_versioning.py

```diff
@@ -0,0 +1,54 @@
+"""sqlalchemy_versioning
+
+Revision ID: 4039aa74290f
+Revises: 
+Create Date: 2022-07-20 14:44:38.393158
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import biblib.models
+
+# revision identifiers, used by Alembic.
+revision = '4039aa74290f'
+down_revision = None
```

Comment:
Once you consolidate with the `alembic/` folder, this will need to be regenerated so the downgrade works too